### ISSUE TITLE
Refill Chests converted to Protector Chests don't work properly

### ISSUE THREAD

**Issue by dark22** [CoderCow/Protector-Plugin#50]
Chests converted for Protector chests does not work properly with Refill Chests. After conversion items from the chest not return as expected. Is there any possibility to be fixed?

I looked for a way to make a small donation to you, but not found. If you still accept donations, let me know. Thanks.

**Comment by CoderCow** [comment]
So I suppose the steps to reproduce this are: 
- place down a chest, 
- put items into it, 
- convert it into a Protector chest, 
- make it a refill chest, 
- take items out of it with a non-owning player
- item will not be refilled

Did you put the items into the chest before you converted it / made it a refill chest or after that? If the owner of the refill chest changes the content, like adding a new item to it, will the problem persist?

I'm not taking donations anymore, thank you.

**Comment by dark22** [comment]
I made a clean install with the latest stable version of TShock without additional plugins using only the latest version of Protector in a new map.

**Login as superadmin and:**
- Place a new chest (It was automatically protected)
- I put some items
- Convert the chest in Protector chest
- Make a refill chest without additional arguments (/rchest)

**Login as user account and:**
- I picked up the items in chest
- Items will not be refilled

If you consider in a future donations, let me know. I do not want to do it for updates, but use your plugin for a long time so I would like to thank in some way.
